devicon.json

```
@@ -3126,6 +3126,26 @@
         "color": "#3FB6D3",
         "aliases": []
     },
+    {
+        "name": "forgejo",
+        "altnames": [],
+        "tags": [
+            "software"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark",
+                "plain",
+                "plain-wordmark"
+            ],
+            "font": [
+                "original"
```

A little whoopsie that needs to be fixed ;)
```suggestion
                "plain",
                "plain-wordmark"
```